### ISSUE TITLE
Typo in https://cs231n.github.io/neural-networks-case-study/

### ISSUE THREAD

**Issue by dementrock** [cs231n/cs231n.github.io#45]
In section "Compute the loss":

```python
corect_logprobs = -np.log(probs[range(num_examples),y])
```

and

```python
# compute the loss: average cross-entropy loss and regularization
data_loss = np.sum(corect_logprobs)/num_examples
reg_loss = 0.5*reg*np.sum(W*W)
loss = data_loss + reg_loss
```

`corect_logprobs` => `correct_logprobs`
